elastic/kibana
# [Beats CM] Improve UX for assignment of tag to list of beats

## Changes
The goal here was to provide better feedback to the user when they're assigning a tag to a set of beats.

![sep-04-2018 16-08-23](https://user-images.githubusercontent.com/18429259/45055167-09f07200-b05d-11e8-9719-04b83e7e6ece.gif)

## Testing the Changes
- Enroll several beats
- Create several tags
- Navigate to `/management/beats_management/overview/beats` and select some or all
- Click the _Set tags_ dropdown and choose a tag to assign/unassign
- Ensure that the notification toasts displayed make sense from a grammatical perspective for singular and plural beats
